### PR TITLE
chore: add Provider component as global decorator to DF Storybook

### PR DESCRIPTION
**Pull request by dozortsev**
## 🚀 Changes

Move `Provider` component setup from `JsonSchemaForm` Strorybook to global config and define it as a decorator.

Screen Shot 2021-04-09 at 4 45 14 PM

### CODE REVIEW

File: packages/dynamic-flows/src/jsonSchemaForm/JsonSchemaForm.story.js [hunk 1]
```diff
@@ -31,7 +21,6 @@ export const basic = () => {
   const schemas = {
     simple: simpleSchema,
     promotedOneOf: promotedOneOfSchema,
-    promotedOneOfCheckbox: promotedOneOfCheckboxSchema,
```
Comment (helendurranttw) [inline packages/dynamic-flows/src/jsonSchemaForm/JsonSchemaForm.story.js:34]: Why did you remove this?

Comment (dozortsev) [reply packages/dynamic-flows/src/jsonSchemaForm/JsonSchemaForm.story.js]: fixed https://github.com/transferwise/neptune-web/pull/889/commits/1cd8c397c20399aa691ea659ee46503bdaf72e51

File: packages/dynamic-flows/.storybook/preview.js [hunk 2]
```diff
@@ -45,4 +54,24 @@ const CenterDecorator = (storyFn) => (
   </div>
 );
 
-export const decorators = [StrictModeDecorator, CenterDecorator];
+// list is not exhaustive but should enough for testing diff edge cases
+// feel free to add more during development
+const severalExamplesOfSupportedLocales = [DEFAULT_LOCALE, 'ja-JP'];
```
Comment (StevePole) [inline packages/dynamic-flows/.storybook/preview.js:59]: Could we include the other ones we had before?  Some were chosen as they had some outlier behaviour

    'en-US',
    'ja-JP',
    'zh-HK',
    'es',
    'fr',
    'ru',
    'de',
    'tr',

File: packages/dynamic-flows/.storybook/preview.js [hunk 3]
```diff
@@ -45,4 +54,24 @@ const CenterDecorator = (storyFn) => (
...
+
+const ProviderDecorator = (storyFn) => {
+  const locale = select('locale (global)', severalExamplesOfSupportedLocales, DEFAULT_LOCALE);
+  const lang = getLangFromLocale(locale);
+  // eslint-disable-next-line fp/no-mutating-assign
+  const supportedLangs = Object.assign(
```
Comment (StevePole) [inline packages/dynamic-flows/.storybook/preview.js:65]: Spread?  I'm still paranoid about browsers that don't support Object.assign...

File: packages/dynamic-flows/.storybook/preview.js (hunk 2 above)
Comment (dozortsev) [reply packages/dynamic-flows/.storybook/preview.js:59]: I included only those two because atm DF supports only them https://github.com/transferwise/neptune-web/blob/main/packages/dynamic-flows/i18n/index.js
